src/storage-client/src/client.rs

```diff
@@ -285,17 +300,35 @@ pub enum StorageResponse<T = mz_repr::Timestamp> {
     FrontierUppers(Vec<(GlobalId, Antichain<T>)>),
     /// Punctuation indicates that no more responses will be transmitted for the specified ids
     DroppedIds(Vec<GlobalId>),
+
+    /// A list of statistics updates, currently only for sources
```

Comment:
This is probably just a pet peeve of mine (and I think Dan's), where we try and use proper capitalization and punctuation in comments.

I know the storage code base is not conforming to this, but the persist code base is quite consistent in this. 😅

Reply:
I am trying to get better at this! usually I copy the style from the other variants, which here is lacking, but will upgrade!